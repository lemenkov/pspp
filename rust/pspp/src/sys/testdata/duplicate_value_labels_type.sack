# File header.
"$FL2"; s60 "$(#) SPSS DATA FILE PSPP synthetic test file";
2; 2; 1; 0; -1; 100.0; "01 Jan 11"; "20:53:52"; s64 ""; i8 0 *3;

# Variables.
2; 6; 0; 0; 0x010600 *2; s8 "STR1";
2; 0; 0; 0; 0x050800 *2; s8 "NUM1";

# Duplicate value labels.
3; 1; s8 "xyzzy"; i8 3; s7 "one"; 4; 2; >>1; 1<<;
3; 1; 1.0; i8 3; s7 "one"; 4; 2; >>2; 2<<;

# Character encoding record.
7; 20; 1; 12; "windows-1252";

# End of dictionary.
999; 0;
