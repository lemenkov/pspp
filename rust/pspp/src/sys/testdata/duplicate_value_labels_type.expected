STR1 has duplicate value labels for the following value(s): "xyzzy"

NUM1 has duplicate value labels for the following value(s): 1

╭──────────────────────┬────────────────────────╮
│       Created        │    01-JAN-2011 20:53:52│
├──────────────────────┼────────────────────────┤
│Writer Product        │PSPP synthetic test file│
├──────────────────────┼────────────────────────┤
│       Compression    │SAV                     │
│       Number of Cases│Unknown                 │
╰──────────────────────┴────────────────────────╯

╭─────────┬─╮
│Variables│2│
╰─────────┴─╯

                                               Variables
╭────┬────────┬─────┬─────────────────┬─────┬─────┬─────────┬────────────┬────────────┬──────────────╮
│    │Position│Label│Measurement Level│ Role│Width│Alignment│Print Format│Write Format│Missing Values│
├────┼────────┼─────┼─────────────────┼─────┼─────┼─────────┼────────────┼────────────┼──────────────┤
│str1│       1│     │Nominal          │Input│    6│Left     │A6          │A6          │              │
│num1│       2│     │                 │Input│    8│Right    │F8.0        │F8.0        │              │
╰────┴────────┴─────┴─────────────────┴─────┴─────┴─────────┴────────────┴────────────┴──────────────╯

        Value Labels
╭─────────────────────┬───╮
│Variable Value       │   │
├─────────────────────┼───┤
│str1           xyzzy │one│
├─────────────────────┼───┤
│num1           1     │one│
╰─────────────────────┴───╯
